CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ User-visible changes worth mentioning.
 ## master
 
 - [#PD ID] Your PR short description.
+- [] Allow Doorkeeper::Application#to_json to work without arguments
```

Comment:
Could you please add #1309 here as a PR ID?

Reply:
Sure. I did it.